### ISSUE TITLE
build for Android on macOS failed

### ISSUE THREAD

**Issue by leleliu008** [seladb/PcapPlusPlus#835]
https://github.com/seladb/PcapPlusPlus/archive/v21.11.tar.gz

```
/configure-android.sh \
    --ndk-path /Users/runner/Library/Android/sdk/ndk/23.1.7779620 \
    --target armeabi-v7a \
    --api 21 \
    --libpcap-include-dir /Users/runner/.ndk-pkg/install.d/libpcap/armeabi-v7a/include \
    --libpcap-lib-dir /Users/runner/.ndk-pkg/install.d/libpcap/armeabi-v7a/lib
```
result:
```
******************************************
PcapPlusPlus Android configuration script 
******************************************

Please specify the NDK path using with '--ndk-path'. Exiting...
```

the reason is macOS' getopt command is incompatible with Linux's. macOS' getopt command doesn't support long options.

If you do not wanna rewrite the code about parsing options, user must install `gnu-getopt`

By the way, I suggest you run test on macOS with GithubActions

**Comment by seladb** [comment]
Your PR was merged to `master`. Thank you @leleliu008 for your contribution to this project, I really appreciate it! 🙏 🙏

**Comment by leleliu008** [comment]
It's my pleasure to contribute to such a great project. Also thank you very much for your patience in reviewing such a big change.

**Comment by leleliu008** [comment]
It seems that the PR was not merge into master branch,  https://github.com/seladb/PcapPlusPlus/releases/tag/v22.11 still build failed on macOS.

**Comment by seladb** [comment]
@leleliu008 your PR was merged to `master` a long time ago: https://github.com/seladb/PcapPlusPlus/commit/24eae767e5c6cbd677020da0f50a85dbb8f01812

The build should work on macOS...

**Comment by leleliu008** [comment]
@seladb Sorry, I made a mistake. I still use the version `v21.11`. this PR was already released since `v22.05`
